.github/workflows/_build_test_upload.yml

```diff
@@ -270,11 +270,18 @@ jobs:
           conda activate conda_build_env
           conda install -yq conda-build -c conda-forge
           packaging/build_conda.sh
+          conda index ./conda-bld
       - name: Install TorchData Conda Package
         shell: bash -l {0}
-        run: conda install --offline conda-bld/*/torchdata-*.tar.bz2
+        run: |
+          if [[ ${{ needs.get_release_type.outputs.type }} == 'official' ]]; then
+            CONDA_CHANNEL=pytorch
+          else
+            CONDA_CHANNEL=pytorch-${{ needs.get_release_type.outputs.type }}
+          fi
+          conda install pytorch torchdata cpuonly -c $(pwd)/conda-bld/ -c "$CONDA_CHANNEL"
       - name: Run DataPipes Tests with pytest
-        shell: bash
+        shell: bash -l {0}
```

Comment:
This refers to the tutorial of `setup-miniconda` GHA : https://github.com/marketplace/actions/setup-miniconda#example-1-basic-usage